### PR TITLE
minorty report blog post

### PR DESCRIPTION
**Pull request by riikkanen**

### CODE REVIEW

File: _posts/2017-07-20-minority-report.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: >
+  Minority report
+author: riikkanen
+excerpt: The pictures used in recruitment annoucments show always a partial glimpse of every day life in Solita. A little more in this minority report.
```
Comment (Kauko) [inline _posts/2017-07-20-minority-report.md]: Typo _announcements_. Mut onkohan se edes oikea sana tähän?

File: _posts/2017-07-20-minority-report.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Real People - No Photostock
+
+In my opinion it is great that Solita uses authentic pictures of real employees in these advertisements. Some of the pictures are even shooted in real situations during the work day. At least the one I'm in presents a real case where we were debugging the software with a colleague when the photographer just came out with his camera. I think that picture is used e.g. for recruiting integration specialists. It must be remembered, that the picture chosen in the announcement is always a partial glimpse to the day in life in Solita. I'm not an integration specialist and we were not recruiting especially women.
```
Comment (Kauko) [inline _posts/2017-07-20-minority-report.md]: "Some of the pictures are even shooted..". Shot, ei shooted.

Comment (Kauko) [inline _posts/2017-07-20-minority-report.md]: itse asiassa sen pitäis olla _taken_. "Shoot" taitaa viitata yleensä videokuvaan.

File: _posts/2017-07-24-minority-report.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: >
+  Minority report
+author: riikkanen
+excerpt: The pictures used in recruitment advertisements show always a partial glimpse of everyday life in Solita. A little more in this minority report.
```
Comment (annisall) [inline _posts/2017-07-24-minority-report.md]: Pitäisikö sanajärjestystä hieman muokata? "The pictures used in recruitment advertisements show always a partial glimpse of everyday life in Solita."
=> "The pictures used in recruitment advertisements always show a partial glimpse of everyday life in Solita."

File: _posts/2017-07-24-minority-report.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+![Debugging a software](/img/minority-report/debugging.jpg)
+*Debugging a software*
+
+## Working as a Software Developer
+
+I've worked as a software developer in Solita already for ten years. I've had opportunity to work in various projects and with several co-workers. It's true that women are few in software development. I just calculated that I've had a female fellow-programmer only in one project but maybe in compensation there were not only one but two of them in that project. The minority of female developers sticks out every autumn, on Solita developer days in Sappee, where we have nine cottage rented for all of us, and only one of them is dedicated for female developers.
```
Comment (annisall) [inline _posts/2017-07-24-minority-report.md]: nine cottage => nine cottages

File: _posts/2017-07-24-minority-report.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+According to my experience, the working with a female software developer is like working with a male software developer. Furthermore, I suppose that working as a female software developer is similar as working as a male software developer. My colleague wrote nicely about [being a software developer in Solita](http://dev.solita.fi/2017/03/24/being-a-software-designer-at-solita.html).
+
+I believe that Solita does not make any difference in recruiting male or female employees; the main point is just recruiting brilliant developers. I think that the problem emerges already at school before high school and university. The ability to make choices regardless of gender should be enabled, allowed and also encouraged. At least in my days in the Helsinki University of Technology, there were lack of female students in certain branches of science, e.g. information technology and electrical engineering.
```
Comment (annisall) [inline _posts/2017-07-24-minority-report.md]: There were lack of female students => there was lack of female students

**Pull request merged**